eks-anywhere-common/Addons/Partner/dynatrace/dynatrace-operator/dynatrace-release.yaml

```diff
@@ -0,0 +1,17 @@
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: dynatrace-operator
+  namespace: dynatrace
+spec:
+  chart:
+    spec:
+      chart: dynatrace-operator
+      sourceRef:
+        kind: HelmRepository
+        name: dynatrace-operator
+        namespace: flux-system
+      version: 0.10.1
+  interval: 1m0s
```

Comment:
How do we pass the credentials and/or other sensitive info to make it work? E.g. license key, etc.